src/components/Home.vue

```diff
@@ -88,38 +85,51 @@ export default {
     this.loaded = true
   },
   computed: {
-      needs(){
-        return this.$store.state.needs
-      },
-      userName(){
-        if (this.$store.state.users[0]) {
-          return this.$store.state.users[0].firstName
-        }
-        else {
-          return ''
-        }
-      },
-      userStatus(){
-        if (this.$store.state.users[0]) {
-          return this.$store.state.users[0].status
-        }
-        else {
-          return ''
-        }
-      },
-      entries(){
-        if (this.$store.state.entries) {
-          let weekdays = this.$store.state.entries[0].sleep.map(a => a.weekday);
-          let hrsSlept = this.$store.state.entries[0].sleep.map(a => a.hrsSlept);
-          console.log('entries', this.$store.state.entries)
-          //console.log('weekdays', hrsSlept)
-          return this.$store.state.entries
-        }
-        else {
-          return ''
+    needs(){
+      return this.$store.state.needs
+    },
+    userName(){
+      if (this.$store.state.users[0]) {
+        return this.$store.state.users[0].firstName
+      }
+      else {
+        return ''
```

Comment:
Semicolon

Alternatively for this block you can write:

`return this.$store.state.users[0] ? this.$store.state.users[0].firstName : '';`

(it's a shortcut for if else blocks)